assets/controllerConfig/configuration.js

```diff
@@ -1,3 +1,3 @@
 export default {
-  baseUrl: process.env.REACT_APP_BASE_URL || ''
-}
+  baseUrl: 'http://localhost:8030',
```

Comment:
Revert this change
 or 
change to `  baseUrl: process.env.PUBLIC_URL || ''` and set env var locally.

Changing it in repo will break deployment in the future.